src/index.ts

```diff
@@ -26,7 +28,13 @@ export const main = async (): Promise<void> => {
     TE.fromOption(() => new Error("Jira ID not found in PR title")),
     TE.chain(getJiraIssues),
     TE.map(RA.map(fromJiraToGenericTicket)),
-    TE.bimap((err) => warn(err.message), renderTickets)
+    TE.bimap(
+      (err) => warn(err.message),
+      (tickets) => {
+        renderTickets(tickets);
```

Comment:
Is it intentional not to use the function's output?
Are there any side effects?

Reply:
`warn` is a side effect